### PR TITLE
Docs Landing page

### PR DESCRIPTION
**Pull request by Himangini**
### Description
Landing page for eksctl documentation

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [x] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: userdocs/src/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+```sh
+eksctl create cluster --version=1.24
+```
+
+### Listing clusters
+
+To list the details about a cluster or all of the clusters, use:
+
+```sh
+eksctl get cluster [--name=<name>][--region=<region>]
```
Comment (a-hilaly) [inline userdocs/src/getting-started.md:117]: My favorite flag is `--all-regions` :)

File: userdocs/src/javascripts/extra.js
Comment (a-hilaly) [inline userdocs/src/javascripts/extra.js:1]: Nice!

**Review by a-hilaly — APPROVED** [review]
Neat and easy to read - thank you Gini!

I'm wondering is there a command that i can run to preview the website, or should I just open the htmls?

File: userdocs/src/installation.md [hunk 2]
```diff
@@ -351,6 +159,7 @@ system settings.
 eksctl completion powershell > C:\Users\Documents\WindowsPowerShell\Scripts\eksctl.ps1
 ```
 
+<!-- Todo: Move features to homepage-->
```
Comment (cPu1) [inline userdocs/src/installation.md:162]: Is this TODO meant to be resolved in this PR?

**Review by cPu1 — APPROVED** [review]

File: userdocs/src/installation.md (hunk 2 above)
Comment (Himangini) [reply userdocs/src/installation.md:162]: this will be a future iteration, we need to come up with a design for this

**Comment by Himangini** [comment]
> Neat and easy to read - thank you Gini!
> 
> I'm wondering is there a command that i can run to preview the website, or should I just open the htmls?

@a-hilaly you can see it in action here https://deploy-preview-7092--eksctl.netlify.app/ ✨ 
Screenshot 2023-09-29 at 11 44 41

**Pull request merged**